samples/nrf9160/modem_shell/src/rest/rest_shell.c

```diff
@@ -0,0 +1,301 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdlib.h>
+#include <zephyr.h>
+#include <assert.h>
+
+#include <shell/shell.h>
+#include <getopt.h>
+
+#include <net/http_parser.h>
+#include <net/rest_client.h>
+
+#include "rest_shell.h"
+#include "mosh_print.h"
+
+static const char rest_shell_cmd_usage_str[] =
+	"Simple REST shell client\n"
+	"Usage:\n"
+	"rest -d host [-m method] [-p port] [-s sec_tag] [-v verify_level] [-u url] [-t timeout_in_ms]\n"
+	"               [-H header1] [-H header2 ... header10] [-b body] [-l response_buffer_length]\n"
```

Comment:
`[-H header2 ... header10]`
This gives the impression that you can put several headers for an argument of one -H option. Is that the intention? If not, I would probably just put (without knowing if there is really a common practice to indicate how many times a parameter can be given):
`[-H header1] [-H header2] ... [-H header10]`

Reply:
Yes, that was NOT an intention. A good point, will change that.